class7-week3/index.js

```diff
@@ -5,11 +5,23 @@ const app = Express()
 
 app.use(bodyParser.json())
 
-const {list, create, update, remove} = require('./actions')
+const {
+    list,
+    create,
+    update,
+    remove,
+    clear,
+    markAsNotDone,
+    markAsDone
+} = require('./actions')
 
 app.get('/todos', list)
 app.post('/todos', create)
 app.put('/todos/:id', update)
 app.delete('/todos/:id', remove)
+app.delete('/todos', clean)
```

Comment:
so you know, I can't run your code without making this change myself. you should never submit a PR that doesn't run. this results in "breaking the build" - something you don't want to do.